app/services/platform-apps/index.ts

```diff
@@ -73,11 +73,26 @@ interface IAppManifest {
   authorizationUrls: string[];
 }
 
+interface IProductionApp {
```

Comment:
I wonder if we should call this interface `IProductionAppResponse` or something, just to make it obvious this is a raw server response.